dataland-keycloak/dataland_theme/login/src/passwordStrength.ts

```diff
@@ -0,0 +1,99 @@
+import { zxcvbn, zxcvbnOptions, debounce } from '@zxcvbn-ts/core'
+import zxcvbnCommonPackage from '@zxcvbn-ts/language-common'
+import zxcvbnEnPackage from '@zxcvbn-ts/language-en'
+
+zxcvbnOptions.setOptions({
+    translations: zxcvbnEnPackage.translations,
+    graphs: zxcvbnCommonPackage.adjacencyGraphs,
+    dictionary: {
+        ...zxcvbnCommonPackage.dictionary,
+        ...zxcvbnEnPackage.dictionary,
+    },
+})
+
+const passwordFieldId = document.querySelector<HTMLInputElement>('#password-new') ? "password-new" : "password"
+
+const passwordField = document.querySelector<HTMLInputElement>(`#${passwordFieldId}`)!
+const confirmPasswordField = document.querySelector<HTMLInputElement>('#password-confirm')!
+const form = document.querySelector<HTMLInputElement>('form')!
+const passwordStrengthIndicatorInner = document.querySelector<HTMLInputElement>('#password-strength-indicator > div')!
+
+function overrideFieldErrorMessage(fieldId: string, errorMessage?: string) {
```

Comment:
Why not pass a field instead of a fieldId?

Reply:
Fixed